### PR TITLE
Extend get_software_environments_all query to provide also package_extract_run id and datetime.

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Related-To: https://github.com/thoth-station/user-api/issues/1543

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -2799,6 +2799,7 @@ def get_hardware_environments_all(
     def get_software_environments_all(
         self,
         is_external: bool = False,
+        include_ids: bool = False,
```
Comment (fridex) [inline thoth/storages/graph/postgres.py:2802]: Do we use this query elsewhere which requires keeping the API strictly backward compatible?

Comment (pacospace) [reply thoth/storages/graph/postgres.py:2802]: No to my knowledge 🤔 should we make it default choice?

Comment (pacospace) [reply thoth/storages/graph/postgres.py:2802]: Maybe metrics-exporter

Comment (fridex) [reply thoth/storages/graph/postgres.py:2802]: I did find it used in metrics-exporter. I think we can safely just extend the dictionary returned with additional values introduced - no need to add `include_ids` parameter.

Comment (pacospace) [reply thoth/storages/graph/postgres.py:2802]: > Maybe metrics-exporter

ok no, we are good

Comment (pacospace) [reply thoth/storages/graph/postgres.py]: > I did find it used in metrics-exporter. I think we can safely just extend the dictionary returned with additional values introduced - no need to add `include_ids` parameter.

Adjusted, thanks!!

File: thoth/storages/graph/postgres.py [hunk 2]
```diff
@@ -2842,8 +2868,49 @@ def get_software_environments_all(
...
+                        "cuda_version": r[0],
+                        "datetime": r[12],
+                        "env_image_name": r[1],
+                        "env_image_tag": r[2],
+                        "environment_name": r[3],
+                        "environment_type": r[4],
+                        "image_sha": r[5],
+                        "os_name": r[6],
+                        "os_version": r[7],
+                        "package_extract_run_id": r[11],
```
Comment (fridex) [inline thoth/storages/graph/postgres.py:2907]:
```suggestion
                        "package_extract_document_id": r[11],
```

**Review by fridex — COMMENTED** [review]
One minor comment, otherwise lgtm 👍🏻

**Review by fridex — APPROVED** [review]
lgtm, thanks! 💯 👍🏻

**Pull request merged**